src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -185,3 +186,232 @@ If you publish your template as *Unlisted*, you can test it under exactly the sa
+
+==== Remix
+
+Selecting *Remix* displays a special remixing page, if you defined one in your template.
+Otherwise, it displays the editable elements of your template and allows you to change them.
+A separate pane displays the *VCC Values* as they appear in the code for the template.
+Events associated with the *Remix* context are displayed in an event pane.
+See <<_viewing_events>>.
+
+*Remix* gives you a dynamic view of remix events, for a better understanding of how VCC data is changing over time in response to user actions.
+
+==== Thumbnail
+
+Selecting the *Thumbnail* shows you exactly how your template will look if you share it on social media.
+See <<rich-preview-image#>>.
+
+*Thumbnail* shows previews in both square and rectangular format.
+
+==== Admin
+
+If the admin context is enabled, selecting *Admin* displays the admin page.
+Events associated with the admin context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== About
+
+If the about context is enabled, clicking *About* displays the page that appears when the user clicks *Remix this Koji*.
+Events associated with the about context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== Sticker
+
+The *Sticker* context shows you how your template behaves when it's embedded inside another template.
+Events associated with the sticker context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== Dynamic Receipt
+
+If a dynamic receipt is enabled, a page is displayed for editing it.
+See <<withkoji-koji-iap-package#, in-app purchases>>.
+
+=== Viewing events
+
+The template generates events that are sent to the Koji platform, and the Koji platform generates events that are sent to the template.
+In most cases, selecting one of the contexts in the debugger displays an events pane which shows the events associated with that context.
+An events pane allows you to see in real time the events that are generated as you interact with the template in the selected context.
+Events generated by the template are listed on the right side of the events pane.
+Events generated by the Koji platform are listed on the left side.
+
+The info for each event includes:
+
+* The type of event – PLATFORM or TEMPLATE
+* The name of the event – For example: *Checked*
+* Properties set by the event (optional) – This may include descriptions of the properties.
+* New values for the properties (optional) – The new values for the properties set by the event.
+
+If you open a template and select the *Remix* context, here's an example of some events that you might see in the events pane:
+
+* TEMPLATE +
+Ready
+
+* PLATFORM +
+Changing remix state +
+VALUE +
+Remixing
+
+* TEMPLATE +
+Set value +
+PATH +
++++[general, title]+++ +
+NEW VALUE +
++++""+++
+
+If you click in an editable field on the remixing page, a *Clicked* event will appear in the event pane.
+For example:
+
+* TEMPLATE +
```

Comment:
I'm not sure about using a bulleted list to format these example events. Maybe there's another way to call them out? A code or an example block, perhaps?